src/actions/task-definitions.js

```diff
@@ -29,7 +29,7 @@ const commandKey = () => (OSName === 'MacOS' ? '⌘' : 'Ctrl')
 const tasks = {}
 
 tasks.evaluateCell = new UserTask({
-  title: 'Evaluate Cell',
```

Comment:
is there a reason we're moving from "evaluate" to "run" here?

Reply:
couple reasons:
- jupyter uses the "run" terminology
- "run" fits better in menus and tooltip
- it's a little less wordy

fwiw: i think this is mostly for user-facing stings, and that using "evaluate" within the iodide code base is good